webpack.common.js

```diff
@@ -1,6 +1,10 @@
 const path = require("path");
+const { CleanWebpackPlugin} = require("clean-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 
+// assets.js
+const Assets = require('./assets');
```

Comment:
Should we just inline the assets here? I can't think of an advantage to having them in a separate file.

Reply:
@thornbill you say like this 
`const Assets = [
    'alameda/alameda.js',
    'requirejs/require.js'
];`

Reply:
Yeah I prefer that personally.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/549/commits/863eb067c22b45a1416800d3dc2667d896dbf12a